File: 2019/territoriali/party/solutions/sol_from_stub.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    // le seguenti due righe possono essere utili per leggere e scrivere da file

    //freopen("input.txt", "r", stdin);
    //freopen("output.txt", "w", stdout);

    int T;
    cin >> T;

    for (int t = 1; t <= T; t++) {
        int N;
        cin >> N;

        // scrivi in questa variabile la soluzione
        int sol = 0;

        for (int i = 0; i < N; i++) {
            int A;
            cin >> A;

            // il valore A rappresenta il valore di amicizia dell'i-esimo amico
            sol += A > 0 ? A : 0;
        }

        cout << "Case #" << t << ": " << sol << endl;
    }
}

File: 2019/territoriali/party/solutions/sol_from_stub.pas
var
    T, test, N, i, sol, A: longint;

begin
    (* le seguenti due righe possono essere utili per leggere e scrivere da file *)

    (*
    assign(input, 'input.txt'); reset(input);
    assign(output, 'output.txt'); rewrite(output);
    *)

    readln(T);

    for test := 1 to T do
    begin
        (* scrivi in questa variabile la soluzione *)
        sol := 0;

        readln(N);

        for i := 1 to N do
        begin
            read(A);

            if A > 0 then
                inc(sol, A);
        end;
        readln;

        writeln('Case #', test, ': ', sol);
    end;
end.
